cypress/support/commands.js

```diff
@@ -30,33 +30,40 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy^="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  });
-});
+Cypress.Commands.add(
+  'register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/api/users', {
+      user: {
+        email,
+        username,
+        password,
+      },
+    });
+  }
+);
 
-Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/api/users', {
-    user: {
-      email,
-      username,
-      password
-    }
-  }).then(response => {
-    const user = {
-      bio: response.body.user.bio,
-      effectiveImage: "https://static.productionready.io/images/smiley-cyrus.jpg",
-      email: response.body.user.email,
-      image: response.body.user.image,
-      token: response.body.user.token,
-      username: response.body.user.username,
-    };
-    window.localStorage.setItem('user', JSON.stringify(user));
-    cy.setCookie('auth', response.body.user.token);
-  });
-});
+Cypress.Commands.add(
+  'login',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/api/users/login', {
+      user: {
+        email,
+        username,
+        password,
+      },
+    }).then((response) => {
+      const user = {
+        bio: response.body.user.bio,
+        effectiveImage:
+          'https://static.productionready.io/images/smiley-cyrus.jpg',
+        email: response.body.user.email,
+        image: response.body.user.image,
+        token: response.body.user.token,
+        username: response.body.user.username,
+      };
+      window.localStorage.setItem('user', JSON.stringify(user));
+      cy.setCookie('auth', response.body.user.token);
+    });
+  }
+);
```

Comment:
The 'login' command includes setting a cookie named 'auth', but it might be beneficial to verify that the login request was successful before setting the cookie. Consider adding assertions to check the response status or body.